# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/extensions/ExtensionPopups.jsm
# Commit: b294575c279c
# Full Hash: b294575c279c9681d0a26b1592f47014262f1d03
# Author: Alexis Beingessner <a.beingessner@gmail.com>
# Date: 2019-07-12 09:56:21
# Regressor Bug: 1547351
# File Overlap Count: 2
# Description:
#   Bug 1547351 - Automatically set content process render root r=kats
#   
#   This implements the idea of automatically setting a content proc's
#   render root based on the render root enclosing the iframe that
#   points to it. There was a bit of cleanup in here that was a bit
# ==============================================================================

diff -r 69260373f95c -r b294575c279c browser/components/extensions/ExtensionPopups.jsm
--- a/browser/components/extensions/ExtensionPopups.jsm	Fri Jul 12 03:19:28 2019 +0000
+++ b/browser/components/extensions/ExtensionPopups.jsm	Fri Jul 12 03:19:28 2019 +0000
@@ -293,7 +293,6 @@
     if (this.extension.remote) {
       browser.setAttribute("remote", "true");
       browser.setAttribute("remoteType", E10SUtils.EXTENSION_REMOTE_TYPE);
-      browser.setAttribute("renderroot", "content");
     }
 
     // We only need flex sizing for the sake of the slide-in sub-views of the